.github/workflows/release.yml

```diff
@@ -51,28 +51,33 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
-      
+
       # Install & Build
       - name: yarn install
         run: |
           yarn config set network-timeout 300000
           yarn install --prefer-offline
+        env:
```

Comment:
Do we need these env vars here? 🤔

Reply:
example https://github.com/spacemeshos/smapp/runs/6155122648?check_suite_focus=true